sw-emulator/lib/periph/src/helpers.rs

```diff
@@ -23,7 +23,9 @@ pub trait U32Array: AsRef<[u32]> {
 }
 pub trait U32ArrayBytes: AsRef<[u8]> {
     const WORD_LEN: usize;
-    type WordArray: AsMut<[u32]> + Default;
+    type WordArray: AsMut<[u32]>;
+    // Required because Default isn't implemented for [u32; 64 and higher]
+    fn default_result() -> Self::WordArray;
```

Comment:
nit: why not have `impl Default for [u8; $word_len * 4]` in the macro?

Not blocking for this PR. If this ends up the only comment I'll merge anyway.